app/src/main/java/com/amaze/filemanager/asynchronous/asynctasks/LoadFilesListTask.java

```diff
@@ -117,6 +117,7 @@ public LoadFilesListTask(
     }
 
     HybridFile hFile = null;
+    MainFragmentViewModel mainFragmentViewModel = mainFragment.getMainFragmentViewModel();
```

Comment:
Nullable, please do necessary checks.

Reply:
It's already checked in line 114, it can never be null here.

Reply:
You should use the same instance that was checked against null for better portability to Kotlin.

Reply:
Fixed